src/haz3lcore/dynamics/elaborator.re

```diff
@@ -86,6 +86,7 @@ let rec dhexp_of_uexp = (m: Statics.map, uexp: Term.UExp.t): option(DHExp.t) =>
         // placeholder logic: sequence
         tl |> List.fold_left((acc, d) => DHExp.Sequence(d, acc), hd) |> wrap
       };
+    | Undefined => Some(DHExp.EmptyHole(u, 0))
```

Comment:
Let's add an undefined to DHExp as well so results are a bit more comprehensible.